stable/percona/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: percona
-version: 1.1.0
+version: 1.1.1
```

Comment:
```suggestion
version: 1.2.0
```

Reply:
Thanks @davidkarlsen . I had some trouble with company sso over github ui, so committed the same fix.